week-1/G-strings-methods/exercise.js

```diff
@@ -1,3 +1,3 @@
 // Start by creating a variable `message`
-
-console.log(message);
+name = 'Jack';
```

Comment:
Same as above.

Reply:
Mohammad, can you fix all your variables as requested by Martyna?